semantic_model_generator/data_processing/cte_utils.py

```diff
@@ -77,18 +80,25 @@ def remove_ltable_cte(sql_w_ltable_cte: str) -> str:
     return sql_without_logical_cte  # type: ignore [no-any-return]
 
 
+def _get_col_expr(column: semantic_model_pb2.Column) -> str:
+    """Return column expr in SQL format.
+    Raise errors if columns is of OBJECT_DATATYPES, which we do not support today."""
```

Comment:
let's make sure spaces aren't in the name

Reply:
done